Makefile

```diff
@@ -74,51 +75,61 @@ __check_defined = \
 ## End-to-end (E2E) Testing ##
 ##############################
 
+# Include project name in image name so that image name
+# does not conflict with other images during local development
+E2E_IMAGE_NAME := $(PROJECT_ROOT)-$(APP_NAME)-e2e
+
 e2e-build: ## Build the e2e Docker image, if not already built, using ./e2e/Dockerfile
-	docker build -t playwright-e2e -f ./e2e/Dockerfile .
+	docker build -t $(E2E_IMAGE_NAME) -f ./e2e/Dockerfile .
 
-e2e-clean-report: ## Remove the local ./e2e/playwright-report and ./e2e/test-results folder and their contents
+e2e-clean-report: ## Remove the local e2e report folders and content
 	rm -rf ./e2e/playwright-report
 	rm -rf ./e2e/blob-report
 	rm -rf ./e2e/test-results
+	rm -rf ./e2e/example-screenshot.png
 
 e2e-delete-image: ## Delete the Docker image for e2e tests
```

Comment:
Didn't notice this target before but I believe the convention for cleanup type tasks is to use the word "clean" e.g. `make clean` so in this case it'd be `make e2e-clean-image` like you have for `make e2e-clean-report`. And I think it'd probably be better to add a `make e2e-clean` that is a convenience for both of them

Reply:
Updated